services/common/src/interfaces/NoWDocument.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { IMineDocument } from "@/interfaces/mineDocument.interface";
```

Comment:
Not something that really needs to be changed, but all of these imports can be `from @/index`

Reply:
The interface files have been adjusted now